code/modules/mob/living/simple_animal/hostile/ordeal/indigo.dm

```diff
@@ -168,3 +172,25 @@
 		L.apply_damage(((pulse_damage + distance - 10)*0.5), BLACK_DAMAGE, null, L.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
 
 
+
+/// cannibalized from wendigo
+/mob/living/simple_animal/hostile/ordeal/indigo_midnight/proc/aoe(range, delay)
+	var/turf/orgin = get_turf(src)
+	var/list/all_turfs = RANGE_TURFS(range, orgin)
+	for(var/i = 0 to range)
+		for(var/turf/T in all_turfs)
+			if(get_dist(orgin, T) > i)
+				continue
+			playsound(T,'sound/effects/bamf.ogg', 600, TRUE, 10)
```

Comment:
You like it loud, huh? _LOUD MA_**KE IT LOUDER, HUH?**

Reply:
FUCK